src/main/java/de/tum/cit/aet/artemis/communication/domain/ForwardedMessage.java

```diff
@@ -25,7 +27,8 @@ public class ForwardedMessage extends DomainObject {
 
     @NotNull
     @Enumerated(EnumType.STRING)
-    @Column(name = "source_type", nullable = false)
+    @JdbcType(PostgreSQLEnumJdbcType.class)
+    @Column(name = "source_type", nullable = false, columnDefinition = "post_type_enum")
```

Comment:
This is not enough to tell Hibernate the name of the enum in PostgreSQL. When it is part of a JPA query, it still tries to find a `postingtype` rather than the `posting_type_enum`.